packages/common/src/utils/urlFormatting.ts

```diff
@@ -137,7 +135,7 @@ export const determinePath = ({ type, contentId }: { type: AppMenuType | undefin
     case APP_CONFIG_ITEM_TYPE.content_list:
       return contentListURL(contentId);
     case APP_CONFIG_ITEM_TYPE.media:
-      return singleMediaURL(contentId);
+      return mediaURL({ id: contentId });
```

Comment:
We could optionally use the label as title here (`Layout.tsx` -> navItems -> label)